test/expect/TestSparseCPU.test_print_cpu_float64.expect

```diff
@@ -46,25 +46,23 @@ tensor(indices=tensor([], size=(0, 10)),
 tensor([], size=(0, 10), dtype=torch.int64)
 # _values
 tensor([], size=(10, 0), dtype=torch.int32)
-########## torch.float32 ##########
+########## torch.float64 ##########
 # sparse tensor
 tensor(indices=tensor([], size=(0, 10)),
        values=tensor([], size=(10, 0)),
-       size=(0,), nnz=10, dtype=torch.float32, layout=torch.sparse_coo)
```

Comment:
is the default dtype set to `double`?

Reply:
Yes the default is dtype is double.   I will update the changes related to the tests in a new PR #54572